docs/templates/preprocessing/image.md

```diff
@@ -30,6 +30,10 @@ Generate batches of tensor image data with real-time data augmentation. The data
     - __shear_range__: Float. Shear Intensity (Shear angle in counter-clockwise direction as radians)
     - __horizontal_flip__: Boolean. Randomly flip inputs horizontally.
     - __vertical_flip__: Boolean. Randomly flip inputs vertically.
+    - __output_size__: Tuple. Resize images (tuple of height and width according to dim_ordering)
+    - __position__: One if {"random", "center", "top left"}.
+        defines where to position a image in the resized one
+    - __background__: Float. Background color to be used for resized image
```

Comment:
Whis is a color a float? Why only these three positions? What would the effect of `output_size` (does it resize pictures or just crop them)? These things are unclear from reading the doc.